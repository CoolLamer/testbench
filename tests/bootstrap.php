<?php

require __DIR__ . '/../vendor/autoload.php';

Test\Bootstrap::setup(__DIR__);
