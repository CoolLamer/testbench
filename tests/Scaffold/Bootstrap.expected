<?php

require%a%/vendor/autoload.php';

Testbench\Bootstrap::setup(__DIR__ . '/_temp', function (Nette\Configurator $configurator) {
	$configurator->addParameters([
		//'appDir' => __DIR__ . '/../src',
		//'testsDir' => __DIR__,
	]);

	$configurator->addConfig(__DIR__ . '/tests.neon');
});
